Common Distributions-Distributions.ipynb

```diff
@@ -0,0 +1,930 @@
+{
```

Comment:
<p>For Gamma Poisson, I get that mu sets the parameterization for the gamma which we then use to parametrize the Poisson dist. What does alpha parameterize? The number of draws from each poisson?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>No no, <code>mu</code> is the Poisson rate and <code>alpha</code> is the Gamma parameter. I haven't done it, but my prediction for the plot below is that, for a fixed mu, if you increase alpha, you can get higher draws of actual Poisson rates. For a fixed alpha, if you increase mu, the draws will get shifted to the right but there will be an upper bound to them. </p><p>LMK if this is wrong or unclear. </p><p> </p><p>I think you can use the random method of the PyMC NegBin distribution, as it's also parametrized as mu and alpha.</p><p> </p><p>You can also add this to the bullet points:</p><ul><li>A Poisson random variable whose rate parameter is Gamma distributed</li></ul><p> </p>